Gemfile.lock

```diff
@@ -0,0 +1,107 @@
+GEM
```

Comment:
I am OK with checking in this file.
We just need to document about this file somewhere in the readme.

Reply:
@aliok @secondsun something something, worked on my machine, locking dependencies? https://stackoverflow.com/a/7518215